src/components/dialogs/filter/expert/expert-filter-constants.ts

```diff
@@ -247,4 +251,16 @@ export const fields: Record<string, Field[]> = {
         FIELDS_OPTIONS.P0,
         FIELDS_OPTIONS.Q0,
     ],
+    BATTERY: [
+        FIELDS_OPTIONS.ID,
+        FIELDS_OPTIONS.NAME,
+        FIELDS_OPTIONS.VOLTAGE_LEVEL_ID,
+        FIELDS_OPTIONS.NOMINAL_VOLTAGE,
+        FIELDS_OPTIONS.CONNECTED,
+        FIELDS_OPTIONS.COUNTRY,
+        FIELDS_OPTIONS.MIN_P,
+        FIELDS_OPTIONS.MAX_P,
+        { ...FIELDS_OPTIONS.TARGET_P, label: 'batteryTargetP' },
```

Comment:
No longer need batteryTargetP and batteryTargetQ, use direct targetP and targetQ